contrib/openshift/build_and_deploy.sh

```diff
@@ -4,13 +4,14 @@ set -exv
 REPOSITORY="quay.io/app-sre"
 IMAGE="${REPOSITORY}/clair"
 
-make container-build
-
 GIT_HASH=`git rev-parse --short=7 HEAD`
+
+docker build --build-arg CLAIR_VERSION=${GIT_HASH} -t clair-service:latest .
```

Comment:
```suggestion
git archive HEAD|
docker build -t clair-service:latest .
```

How does something like this sit with you?